bugfixes.asm

```diff
@@ -235,7 +235,7 @@ ParadoxCaveGfxFix:
     CPX.w #$1E00 : BEQ .uploadLine
 
 .uploadLine
-    LDA.b #$01 : STA.w MDMAEN
+    LDA.b #$01 : STA.w DMAENABLE
```

Comment:
That's an official name of the register in SNES docs: MDMAEN

Reply:
ngl, I hate the register labels and I've mostly decided to not touch them in this PR because that is a whole ordeal. But the labels for them are not coherently labeled, I have no clue what any of them mean when I see them in the code. Personally, I'd rather go back to show $420B rather than the current nonsense, even if it is accurate to some SNES dev manual. Another option is to add the old label back in and just not use it. That way the official label is referenced if someone was somehow confused as to what DMAENABLE means, they could look up the labels and see both of these labels next to each other.

Reply:
Upon reflection, this is probably fine. The name is close enough.